src/styles/products.scss

```
@@ -0,0 +1,23 @@
+$width: calc((200px * 4) + (48px * 3));
+
+.products {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+
+  &__card {
+    margin-left: 48px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &-container {
+      padding: 50px 0 0;
+      max-width: #{width};
```

it's more convenient to use it like that, also easier to look for it in code
```suggestion
      max-width: $width;
```